chp07_CA/Exercise_7_01_WolframCA_randomizedrules/CA.py

```diff
@@ -0,0 +1,92 @@
+# The Nature of Code
+# Daniel Shiffman
+# http://natureofcode.com
+
+# Wolfram Cellular Automata
```

Comment:
I thought this pull request was about chapter 6 examples. Aren't these chapter 7 examples in another pull request?